gtsfm/two_view_estimator.py

```diff
@@ -274,6 +290,8 @@ def aggregate_frontend_metrics(
     inlier_ratio_est_model_all_pairs = []
     num_inliers_gt_model_all_pairs = []
     num_inliers_est_model_all_pairs = []
+    configuration_counts = defaultdict(int)
```

Comment:
Can we use a counter?

Reply:
we could. doesn't matter to me too much. Do you prefer `Counter` to defaultdict?

Reply:
I am fine with this too.